packages/language-server/src/plugins/typescript/TypeScriptPlugin.ts

```diff
@@ -419,4 +424,18 @@ export class TypeScriptPlugin
             this.configManager.enabled(`typescript.${feature}.enable`)
         );
     }
+
+    private getUserPreferences(): ts.UserPreferences {
```

Comment:
Maybe we could put this into the lsandtsdocresolver instead? Everyone already has this as a dependency so we don't need to add a new one. Semantically I also think it makes sense in there. Maybe the getLsAndTsDoc function in there could even return `preferences` as a third object.